packages/database/package.json

```diff
@@ -14,7 +14,7 @@
     "dev": "rollup -c -w",
     "test": "run-p test:browser test:node",
     "test:browser": "karma start --single-run",
-    "test:node": "TS_NODE_CACHE=NO nyc --reporter lcovonly -- mocha 'test/{,!(browser)/**/}*.test.ts' --compilers ts:ts-node/register/type-check -r src/nodePatches.ts --retries 5 --timeout 5000 --exit",
+    "test:node": "TS_NODE_CACHE=NO nyc --reporter lcovonly -- mocha 'test/{,!(browser)/**/}*.test.ts' --compilers ts:ts-node/register/type-check -r src/nodePatches.ts --retries 5 --timeout 5000 --opts ../../config/mocha.node.opts",
```

Comment:
Hrm.  This sets retries / timeout and then pulls in options that are going to set different retries / timeout (and I don't know which one will win).  Same issue exists elsewhere.

FWIW- 3 retries seems excessive to me already. I don't know why we ended up with 5 retries in places...

Reply:
Flags on the command line will override the corresponding ones in mocha.opts, based on the mocha doc [here](https://mochajs.org/#mochaopts). And what I'm trying to propose is to put all default configurations in the config file and only override them with command line flags when necessary.

I have no idea of the 5 retries.

Reply:
Since we created a config file, we should move all the options into the file. It will be easier for people to read.

Reply:
Agreed.  I don't think we actually want to override the defaults in these cases.  The defaults should work for both CI and local development AFAIK.

Reply:
(to be clear, by "defaults" I mean what we put in the config file)